README.md

```diff
@@ -2,61 +2,126 @@
 
 The Substreams:SQL sink helps you quickly and easily sync Substreams modules to a PostgreSQL or Clickhouse database.
 
-### Quickstart
+## Substreams Output Formats
+
+The `substreams-sink-sql` supports two different Substreams output formats, each with distinct advantages:
+
+### Relational Mappings (Recommended)
+
+**How it works:** Tables and rows are extracted dynamically from Protobuf messages using annotations for table and relation mappings. This approach leverages Substreams' built-in relational mapping capabilities.
+
+**Pros:**
+- **Less development work** - No need to manually emit database changes in your Substreams code
+- **Automatic schema inference** - Tables and relationships are derived from your Protobuf definitions
+- **Type safety** - Protobuf annotations ensure data consistency
+- **Easier maintenance** - Schema changes are managed through Protobuf definitions
```

Comment:
Let add also that its faster in most scenarios, due to possibility to perform bulk inserts more easily.